.github/workflows/PR.yml

```
@@ -44,3 +44,18 @@ jobs:
     - name: Test
       run: go test
       working-directory: backend
+      
+  Linting:
+    name: "Linting"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: run linting
+        uses: super-linter/super-linter/slim@v5 
+        env:
+          DEFAULT_BRANCH: ${{ github.event.pull_request.base.ref }}
```

```suggestion
          DISABLE_ERRORS: true
          DEFAULT_BRANCH: ${{ github.event.pull_request.base.ref }}
```
Please disable the errors :)